roles/edpm_network_config/tasks/main.yml

```diff
@@ -36,6 +36,9 @@
         name: NetworkManager
         state: restarted
       when: nm_ovs_status.changed
+    - name: Render network_config variable
+      ansible.builtin.set_fact:
+        network_state: "{{ lookup('template', edpm_network_config_nmstate_template ) }}"
```

Comment:
Should we use the fqcn `ansible.builtin.template` here?

Reply:
You're right. I took the example from the webpage, but in the docs ( `ansible-doc -t lookup template` ) the fqcn is used.

Thanks for spotting @rabi !